pipeline/scripts/average_rating.py

```diff
@@ -0,0 +1,37 @@
+import sys
+
+sys.path.insert(0, "/workspace/butterfly/backend/utils/")
+
+
+def calculate_analytics(matches):
+    totals = dict()
+    count = dict()
+    averages = dict()
+    for match in matches:
+        totals[match["from_user"]] = match["value"] + totals.get(
+            match["from_user"], 0
+        )
+        count[match["from_user"]] = 1 + count.get(match["from_user"], 0)
+    keys = list(totals.keys())
+    for i in range(len(totals)):
+        averages[keys[i]] = totals[keys[i]] / count[keys[i]]
+    return averages
+
+
+def average_rating():
+    from firebase import get_db, initialize_firebase_for_api
+
+    initialize_firebase_for_api()
+    if len(sys.argv) > 1:
+        community = sys.argv[1]
+    else:
+        community = "sds"
```

Comment:
This works great in this case. If you are building Python scripts in the future, you might start to have more arguments and you will want to configure their defaults and make sure that they are set correctly. A library like [argparse](https://docs.python.org/3/library/argparse.html) can be a big help for that!